File: App.Core/Models/Users/CreateUserDto.cs
﻿using System;

namespace App.Core.Models.Users
{
    public class CreateUserDto
    {
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Email { get; set; }
        public int? UserTypeId { get; set; }
        public DateTime? DateOfBirth { get; set; }
        public string Mobile { get; set; }
        public string Address { get; set; }
        public int? ZipCode { get; set; }
        public string ProfileImage { get; set; }
        public int? StateId { get; set; }
        public int? CountryId { get; set; }
    }
}

File: src/app/core/models/interface/User/CreateUserDto.ts
export interface CreateUserDto {
  firstName: string;
  lastName: string;
  email: string;
  userTypeId: number;
  dateOfBirth: string;
  mobile: string;
  address: string;
  zipCode: number;
  profileImage: string;
  stateId: number;
  countryId: number;
}
